--- a/vaudtax-2018
+++ b/vaudtax-2018
@@ -11,11 +11,7 @@
 
 classpath='lib/dvbern-lib-update.jar:lib/cryptutil.jar'
 
-if [ ! -f ///$classpath ] ; then
-	cd `dirname $0`
-else
-	cd //
-fi
+cd /usr/share/vaudtax/
 
 type java &> /dev/null
 if [ "$?" -ne "0" ] ; then
@@ -38,4 +34,4 @@
 fi
 
 
-jre/bin/java -cp $classpath ch.dvbern.lib.update.Launcher
+/usr/bin/java -cp $classpath ch.dvbern.lib.update.Launcher
